.gitignore

```diff
@@ -4,3 +4,4 @@ build*/
 .clangd/
 compile_commands.json
 corpora/
+/.vscode/
```

Comment:
/?

Reply:
this only ignores .vscode in the root folder and not in every folder. I think this is a good idea. 
We could also ignore all .vscode folder, then I would need to rename /contrib/.vscode to /contrib/vscode

Reply:
I'm fine with having the slash here. But I would argue for renaming `/contrib/.vscode` to `contrib/vscode`. Otherwise the directory will be hidden by default which is not what we want, I guess?

Reply:
I renamed it to `vscode`, but left the gitignore as is

Reply:
got it - maybe we want to add some more / in this file :)

Reply:
different PR :)